calico-enterprise_versioned_docs/version-3.19-2/visibility/elastic/flow/datatypes.mdx

```diff
@@ -101,3 +101,66 @@ Where,
     action for the tier.  In this case, the `<policy name>` is selected arbitrarily from the set of policies within
     the tier that apply to the endpoint.
   * `-2` means "unknown". The rule index was not recorded.
+
+### Flow log example, with **no aggregation**
+
+A flow log with aggregation level 0, **`no aggregation`**, might look like:
```

Comment:
```suggestion
A flow log with aggregation level 0, `no aggregation`, might look like:
```
No need for bold when we're already using a code font. Should remove throughout.